src/execute.c

```diff
@@ -1267,7 +1267,8 @@ jv jq_get_prog_origin(jq_state *jq) {
 }
 
 jv jq_get_lib_dirs(jq_state *jq) {
-  return jq_get_attr(jq, jv_string("JQ_LIBRARY_PATH"));
+  jv lib_dirs = jq_get_attr(jq, jv_string("JQ_LIBRARY_PATH"));
+  return jv_is_valid(lib_dirs) ? lib_dirs : jv_array();
```

Comment:
If `lib_dirs` is not valid it might need to be freed nonetheless if it has a message.

Reply:
@nicowilliams No. That is not possible.

Reply:
Oh, because `jq_get_attr()` never does that.  Got it.